src/main/java/edu/greenblitz/tobyDetermined/commands/swerve/MoveByVisionSupplier.java

```diff
@@ -4,6 +4,6 @@
 
 public class MoveByVisionSupplier extends CombineJoystickMovement {
 	public MoveByVisionSupplier(boolean isSlow) {
-		super(isSlow, new AngPIDSupplier(RobotMap.Vision.apriltagLocation.toPose2d().getTranslation()));
+		super(isSlow, new AngPIDSupplier(RobotMap.Vision.redApriltagLocationId1.toPose2d().getTranslation()));
```

Comment:
Raz - What is the use of isSlow? I think we can avoid here passing a boolean to the function.
Maybe we can pass the speed, or have the isSlow as part of the RobotMap?

Reply:
it's not used anymore, it changed when we refactored